.env

```diff
@@ -3,3 +3,11 @@ MYSQL_USER=contenta
 MYSQL_PASSWORD=contenta
 MYSQL_ALLOW_EMPTY_PASSWORD=yes
 MYSQL_ROOT_PASSWORD=root
+
+HOSTNAME=contenta.local
+HOSTIP=192.168.1.1
```

Comment:
I think the final goal would be to have a network set up instead of fixing an IP?

Reply:
Ya, that would likely be a better idea. I was just putting this in as I didn't want it in the docker-compose.yml. Again, the more generic, the better, I suppose.